src/main/java/io/cdap/plugin/gcp/bigquery/sink/BigQuerySinkUtils.java

```diff
@@ -137,7 +139,8 @@ private static void createDataset(BigQuery bigQuery, String dataset, @Nullable S
   private static void createBucket(Storage storage, String bucket, @Nullable String location,
                                    @Nullable String cmekKey, Supplier<String> errorMessage) throws IOException {
     try {
-      GCPUtils.createBucket(storage, bucket, location, cmekKey);
+      GCPUtils.createBucket(storage, bucket, location,
+                            Strings.isNullOrEmpty(cmekKey) ? null : CryptoKeyName.parse(cmekKey));
```

Comment:
It will be fixed in the upcoming PR. As I will be adding cmek as a config in these plugins as well and then they will directly get the CryptoKeyName object. All this happened because I changed the createBucket() parameter to accept CryptoKeyName object instead of string. Is it okay?